src/conf_mode/service_monitoring_prometheus.py

```diff
@@ -57,6 +60,12 @@ def get_config(config=None):
     if tmp:
         monitoring.update({'frr_exporter_restart_required': {}})
 
+    tmp = is_node_changed(conf, base + ['blackbox-exporter', 'vrf']) or is_node_changed(
```

Comment:
Please consolidate this into a for loop
```suggestion
    for node in ['vrf', 'config-file']:
        tmp = is_node_changed(conf, base + ['blackbox-exporter', node])
```